src/app/friend/friend-list/friend-list.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { Friend } from 'src/app/model/friend.model';
+
+@Component({
+  selector: 'app-friend-list',
+  templateUrl: './friend-list.component.html',
+  styleUrls: ['./friend-list.component.scss']
+})
+export class FriendListComponent {
+
+  constructor(private router: Router){};
+
+  //array of friend
+  friends: Friend[] = [
```

Comment:
Inconsistency in methos comments to whole file

Reply:
Changes done.